sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushDownUtils.scala

```diff
@@ -38,9 +37,9 @@ object PushDownUtils extends PredicateHelper {
    * @return pushed filter and post-scan filters.
    */
   def pushFilters(
-      scanBuilder: ScanBuilder,
+      scanBuilderHolder: ScanBuilderHolder,
```

Comment:
why change this?

Reply:
because I need the `scanBuilderHolder.relation` for `DataSourceUtils.getPartitionKeyFiltersAndDataFilters`